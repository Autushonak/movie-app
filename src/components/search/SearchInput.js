import React from 'react';
import styled from 'styled-components';
import { colors } from '../../constants/colors';

const SearchInputStyled = styled.input`
  flex-grow: 10;
  background-color: ${colors.GREY_MAIN};
  color: ${colors.WHITE};
  width: auto;
  opacity: 0.5;
  font-size: 16px;
  padding: 15px 5px;
  border: none;
  border-radius: 4px;
  margin-right: 10px;
`;

const SearchInput = () => <SearchInputStyled />;

export default SearchInput;
