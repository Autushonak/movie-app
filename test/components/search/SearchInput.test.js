import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import SearchInput from '../../../src/components/search/SearchInput';

describe('SearchInput ', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<SearchInput />);
    expect(component).toMatchSnapshot();
  });
});
