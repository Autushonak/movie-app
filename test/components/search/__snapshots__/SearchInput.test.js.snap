// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput  should be rendered correctly 1`] = `
.c0 {
  -webkit-box-flex: 10;
  -webkit-flex-grow: 10;
  -ms-flex-positive: 10;
  flex-grow: 10;
  background-color: #232323;
  color: #FFFFFF;
  width: auto;
  opacity: 0.5;
  font-size: 16px;
  padding: 15px 5px;
  border: none;
  border-radius: 4px;
  margin-right: 10px;
}

<input
  className="c0"
/>
`;
